src/account/saga.ts

```diff
@@ -220,6 +221,25 @@ export function* handleUpdateAccountRegistration() {
   }
 }
 
+export function* handleChooseCreateAccount() {
+  try {
+    yield call([Statsig, 'updateUser'], {
+      custom: { startOnboardingTimestamp: new Date().toISOString() },
+    })
```

Comment:
is this guaranteed to fire immediately? Wondering if we'd run into a scenario where we call this, and immediately request a variant and the targeting gate fails and then later passes.
Also, just to confirm, the latest value is sufficient right in case the user goes back and forth between create / restore flow and this ends up firing multiple times

Reply:
we talked offline and decided to move updateUser to the button click handler to make sure it gets invoked in time

Reply:
> Also, just to confirm, the latest value is sufficient right in case the user goes back and forth between create / restore flow and this ends up firing multiple times

I thought of this too, and opted to keep implementation simpler by just updating the start timestamp each time a user chooses "create account". However, I ran a query just now to see how often a user onboards twice on the same device, and I was surprised to find that the number is actually pretty high, around 8-10% (depending on how you do the filtering; 8% if you exclude users who onboard twice in 30 seconds). This could pose a problem in our experiments, since a user onboarding a second time might behave differently than one onboarding the first time, and we'd likely want to filter them out if our goal is to get "real new users". 

So I'll probly try adding start timestamp to redux to avoid this, and avoid updating start timestamp twice for the same device. 

Unfortunately, I don't think we'll be able to solve this for users onboarding twice on two different devices-- they could get included in experiments, cross bucketed, or the like. But that's out of scope for this PR IMO.